config/manager/manager.yaml

```diff
@@ -36,7 +36,7 @@ spec:
         name: manager
         env:
         - name: FLAGD_VERSION
-          value: INPUT_FLAGD_VERSION
+          value: v0.2.7
```

Comment:
Shouldn't be checked in

Reply:
resolved

Reply:
Any idea how this happens? Can this accidental change be avoided in the future with an update the tooling?

Reply:
I missed this while building the chart for testing, ill look into it to see if we can avoid this happening again